### PR TITLE
Limit message correction

### PR DESCRIPTION
**Pull request by ubbabeck**
# About PR

This pr references issue  https://github.com/breez/breezmobile/issues/1087.

## What is done

- Removed displaying the _maxPaymentAmount_
- Removed unused function _validateOutgoingOnChainPayment_

### CODE REVIEW

**Review by roeierez — APPROVED** [review]
LGTM

File: lib/bloc/account/account_model.dart [hunk 1]
```diff
@@ -403,15 +394,11 @@ class AccountModel {
   String validatePayment(Int64 amount, bool outgoing) {
     final texts = getSystemAppLocalizations();
     if (maxPaymentAmount != null && amount > maxPaymentAmount) {
-      return texts.valid_payment_error_exceeds_the_limit(
-        currency.format(maxPaymentAmount),
-      );
+      return texts.valid_payment_error_exceeds_the_limit("");
```
Comment (ademar111190) [inline lib/bloc/account/account_model.dart:397]: There are better approaches than passing an empty string here.
Removing the variable from the text entry point, you can call the texts without arguments by calling `return texts.valid_payment_error_exceeds_the_limit`

**Review by ademar111190 — CHANGES_REQUESTED** [review]

File: lib/bloc/account/account_model.dart (hunk 1 above)
Comment (ubbabeck) [reply lib/bloc/account/account_model.dart:397]: Regarding you feedback,I was not able to pass a null param so I ended up with generating a new variable without a param.

Is it possible to generate translations that are optional? 

If it is fact is not possible please see this pr https://github.com/breez/Breez-Translations/pull/7

Comment (ademar111190) [reply lib/bloc/account/account_model.dart:397]: Reviewed; thanks for addressing the issue.

File: lib/bloc/account/account_model.dart [hunk 2]
```diff
@@ -269,23 +269,22 @@ class AccountModel {
...
+        posCurrencyShortName ?? _posCurrencyShortName,
+        addedFundsReply: addedFundsReply ?? this.addedFundsReply,
+        onChainFeeRate: onChainFeeRate ?? this.onChainFeeRate,
+        enableInProgress: enableInProgress ?? this.enableInProgress,
+        syncProgress: syncProgress ?? this.syncProgress,
+        syncedToChain: syncedToChain ?? this.syncedToChain,
+        serverReady: serverReady ?? this.serverReady,
+        syncUIState: syncUIState ?? this.syncUIState,
+        initial: initial ?? this.initial,
+        nodeUpgrading: nodeUpgrading ?? this.nodeUpgrading);
```
[the remark this answers is not in the document]
Comment (ubbabeck) [reply lib/bloc/account/account_model.dart]: Added the trailing. Think the dart format is doing its own thing 😕

[the remark this answers is not in the document]
Comment (ademar111190) [reply lib/bloc/account/account_model.dart]: Yeap, it is okay to format the files we are working on, but as we have two valid formatted options here, the one that changes the code less is preferable IMHO; thanks for changing it.

**Review by ademar111190 — APPROVED** [review]

**Pull request merged**